handlers/search.go

```diff
@@ -13,17 +17,67 @@ func Search(w http.ResponseWriter, r *http.Request) {
 	_, err := strconv.Atoi(search)
 
 	if err == nil {
-		http.Redirect(w, r, "/block/" + search, 301)
+		http.Redirect(w, r, "/block/"+search, 301)
 		return
 	}
 
-	search = strings.Replace(search,"0x", "", -1)
+	search = strings.Replace(search, "0x", "", -1)
 
 	if len(search) == 64 {
-		http.Redirect(w, r, "/block/" + search, 301)
+		http.Redirect(w, r, "/block/"+search, 301)
 	} else if len(search) == 96 {
-		http.Redirect(w, r, "/validator/" + search, 301)
+		http.Redirect(w, r, "/validator/"+search, 301)
 	} else {
 		http.Error(w, "Not found", 404)
 	}
-}
\ No newline at end of file
+}
+
+type SearchAheadBlocksResult []struct {
```

Comment:
Please move type definitions into the dedicated types package

Reply:
Moved it to `types/templates.go`